lib/lndclient/LndClient.ts

```diff
@@ -382,19 +395,76 @@ class LndClient extends SwapClient {
   /**
    * Connects to another lnd node.
    */
-  public connectPeer = (pubkey: string, host: string, port: number): Promise<lndrpc.ConnectPeerResponse> => {
+  public connectPeer = (pubkey: string, host: string): Promise<lndrpc.ConnectPeerResponse> => {
```

Comment:
I know lnd is using `host` to refer to both the host but it's a little confusing to me. Maybe we could rename parameter here to `uri` or `address` instead?

Reply:
I changed the param to `address` and the actual request to `lightningAddress`.